web/blog/vast-v3.0/index.md

```diff
@@ -119,11 +120,17 @@ fourth section are `5ec2-7015`:
 }
 ```
 
+:::tip Case-Insensitive Patterns
+In addition to writing `/pattern/`, you can specify a regular expression that
+ignores the casing of characters via `/pattern/i`. The `/i` flag is currently
+the only support pattern modifier.
+:::
+
 ## Revamped Status for Event Distribution
 
-The event distribution statistics moved in the output of `vast status`.
+The event distribution statistics moved within the output of `vast status`.
 
-What previously was available under the `index.statistics` section when using
+This was previously available under the `index.statistics` section when using
```

Comment:
Now too sure about "This" - maybe using "They were..." or "Their location" instead of "This was" to describe the statistics (or the statistics' location) would sound better?

Reply:
Good point; fixed in 62d009ae8.